adoc-mode.el

```diff
@@ -2033,7 +2033,7 @@ START-SRC and END-SRC delimit the actual source code."
             (outer-brackets-and-delimiter (&rest stuff)
                                           (format "^\\[%s\\]\n\\(?2:----+\\)\n"
                                                   (apply #'concat stuff)))
-            (lang () ",\\(?1:[^],]+\\)")
+            (lang () ",[\t ]*\\(source-language[\t ]*=[\t ]*\\)?\\(?1:[^],]+\\)")
```

Comment:
I'd add here some comments with the snippets from the manual, so it's clear what we're matching against. Ideally we should also have some matching unit tests, but this can be done later.

Reply:
Thanks for the review.
The asciidoctor manual is not clear about the 2nd code block attribute. So I had a look at the Ruby code. There I misinterpreted the document attribute as block attribute. With the last commit I corrected this.

Reply:
The rabbit-hole is quite deep: https://github.com/asciidoctor/asciidoctor-pdf/issues/180

Reply:
@bbatsov  ... and even deeper ...
```
:source-highlighter: pygments
:myBlock: source,ruby,subs="verbatim,quotes"


[{myBlock}]
----
class SomeClass
  for i in 1..5 do
      "Hello *world*".print
  end
----
```